stack/stack.go

```diff
@@ -1,39 +1,91 @@
 package main
 
+//Imported Libraries
 import (
-	"errors"
 	"fmt"
 )
 
-// Stack implements stack DS
-type Stack struct {
-	list []int
+//Constants
+const x  = 10
+
+//Implementation of the stack structure
+type stack struct {
+	items [x] string
+}
+//pop from the stack
+func (a *stack) push (value string){
+	var found bool = false
+	for index,valuea := range a.items{
+		if valuea == "" {
+			a.items[index]=value
+			found=true
+			break
+		}
+	}
+	if found == false{
+		fmt.Printf("Stack is full, Try on PUSH something outside\n")
+	}
 }
 
-// Push pushes to stack
-func (s *Stack) Push(data int) error {
-	s.list = append(s.list, data)
-	return nil
+//pop form the stack
+func (a *stack) pop()  {
+	var empty bool = true
```

Comment:
should omit type bool from declaration of var empty; it will be inferred from the right-hand side

*Origin: GoLintBear, Section: `golint`.*